File: CS/Model/File.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DragDropTwoGrids.Model {
    public class File {
        public string FileName { get; set; }
        public long FileSize { get; set; }
        public File(string fileName, long fileSize) {
            FileName = fileName;
            FileSize = fileSize;
        }
    }
}

File: VB/Model/File.vb
Namespace DragDropTwoGrids.Model

    Public Class File

        Public Property FileName As String

        Public Property FileSize As Long

        Public Sub New(ByVal fileName As String, ByVal fileSize As Long)
            Me.FileName = fileName
            Me.FileSize = fileSize
        End Sub
    End Class
End Namespace
